projects/core/src/group-skipper/facade/group-skipper.service.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  Injectable,
+  Renderer2,
+  RendererFactory2,
+  Inject,
+  OnDestroy,
+  OnInit,
+} from '@angular/core';
+import { GroupSkipperConfig, GroupSkipperElement } from '../config';
+import { CmsService, Page } from '../../cms';
```

Comment:
maybe we can improve this import by adding the barrel here, instead of plain `../cms` (`../cms/index`)